manim/utils/hashing.py

```diff
@@ -49,9 +48,7 @@ def reset_already_processed(cls):
 
     @classmethod
     def check_already_processed_decorator(cls: "_Memoizer", is_method=False):
-        """Decorator to handle the arguments that goes through the decorated function.
-        Returns _ALREADY_PROCESSED_PLACEHOLDER if the obj has been processed, or lets
-        the decorated function call go ahead.
+        """Decorator to handle the arguments that goes through the decorated function. Returns _ALREADY_PROCESSED_PLACEHOLDER if the obj has been processed, or lets the decorated function call go ahead.
```

Comment:
```suggestion
        """Decorator to handle the arguments that goes through the decorated function.
        
        Returns _ALREADY_PROCESSED_PLACEHOLDER if the obj has been processed,
        or lets the decorated function call go ahead.
```

This could also be split into a `Returns` section.